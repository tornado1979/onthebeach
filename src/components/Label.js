import React from 'react';
import PropTypes from 'prop-types';

const Label = (props) => {
  return (
    <div className="label">
      <span className={props.type}>{props.text}</span>
    </div>
  )
}

Label.defaultProps = {
  text: 'Iberostar Grand Salome',
  type: 'title'
};

Label.propTypes = {
  text: PropTypes.string,
  type: PropTypes.string
};

export default Label;
