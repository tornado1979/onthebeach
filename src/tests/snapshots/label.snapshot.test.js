import React from 'react';
import renderer from 'react-test-renderer';
import Label from '../../components/Label';

describe('Label', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <Label/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
