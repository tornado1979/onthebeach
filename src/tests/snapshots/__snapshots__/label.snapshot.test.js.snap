// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label renders correctly 1`] = `
<div
  className="label"
>
  <span
    className="title"
  >
    Iberostar Grand Salome
  </span>
</div>
`;
